docs/rules/no-then.md

```diff
@@ -12,26 +12,39 @@ https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/asy
 
 👎 Examples of **incorrect** code for this rule:
 
+```js
+function countData(url) {
+  return downloadData(url).then(data => {
+    return data.length
+  })
+}
+```
+
 ```js
 function getProcessedData(url) {
   return downloadData(url).catch(e => {
     console.log('Error occurred!', e)
+    return null;
   })
 }
 ```
 
 👍 Examples of **correct** code for this rule:
 
+```js
+async function countProcessedData(url) {
+  const data = await downloadData(url);
+  return data.length
+}
```

Comment:
Oh no! now we lost the closing fence 🤦 

```suggestion
```js
async function countProcessedData(url) {
  const data = await downloadData(url);
  return data.length
}
```
```